setup/setup.ui

```diff
@@ -1318,13 +1330,96 @@
+                        <child>
+                          <object class="GtkGrid" id="table3">
+                            <property name="visible">True</property>
+                            <property name="can_focus">False</property>
+                            <property name="column_spacing">12</property>
+                            <property name="row_spacing">6</property>
+                            <child>
+                              <object class="GtkCheckButton" id="checkbutton_custom_theme">
+                                <property name="label" translatable="yes">Use custom theme:</property>
+                                <property name="use_action_appearance">False</property>
+                                <property name="visible">True</property>
+                                <property name="can_focus">True</property>
+                                <property name="receives_default">False</property>
+                                <property name="use_action_appearance">False</property>
+                                <property name="use_underline">True</property>
+                                <property name="halign">start</property>
+                                <property name="draw_indicator">True</property>
+                                <property name="hexpand">True</property>
+                              </object>
+                              <packing>
+                                <property name="left_attach">0</property>
+                                <property name="top_attach">0</property>
+                              </packing>
+                            </child>
+                            <child>
+                              <object class="GtkComboBox" id="combobox_custom_theme">
+                                <property name="visible">True</property>
+                                <property name="can_focus">False</property>
+                                <property name="model">model_custom_theme</property>
+                                <property name="hexpand">True</property>
+                                <child>
+                                  <object class="GtkCellRendererText" id="renderer4"/>
+                                  <attributes>
+                                    <attribute name="text">0</attribute>
+                                  </attributes>
+                                </child>
+                              </object>
+                              <packing>
+                                <property name="left_attach">1</property>
+                                <property name="top_attach">0</property>
+                              </packing>
+                            </child>
+                            <child>
+                              <object class="GtkCheckButton" id="checkbutton_custom_icon">
+                                <property name="label" translatable="yes">Use custom icon:</property>
```

Comment:
Please add "tooltip_text" property of "Choose a theme of the arrow buttons on the candidate window"